### PR TITLE
Adds Users Endpoints

### PR DESCRIPTION
**Pull request by AJLambda**
# Description

Finalizes CRUD operations for users and includes Get User Posts endpoint

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Change Status

- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [x] Tested with Insomnia

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

**Review by Mrsteveson — APPROVED** [review]
endpoints working as intended currently. May need to adjust once authorization protocols are implemented but great work!

File: bash.exe.stackdump [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Stack trace:
+Frame        Function    Args
+000FFFFA358  0018006021E (00180252DED, 001802340A6, 000FFFFA358, 000FFFF9250)
+000FFFFA358  00180048859 (00000000000, 00000000000, 00000000000, 001005D1D3C)
+000FFFFA358  00180048892 (00180252EA9, 000FFFFA208, 000FFFFA358, 00000000000)
+000FFFFA358  001800AF0D8 (00000000000, 00000000000, 00000000000, 00000000000)
+000FFFFA358  001800AF25D (000FFFFA370, 00000000000, 00000000000, 00000000000)
+000FFFFA5E0  001800B0673 (000FFFFA370, 00000000000, 00000000000, 00000000000)
+End of stack trace
```
Comment (KevinBrack) [inline bash.exe.stackdump]: Is this an intended change? Please commit your files individually, and stash anything you do not intend to change. Delete this file on your end, and make a new commit please. That will automatically be added to this PR. Let me know if you need help.

**Review by KevinBrack — CHANGES_REQUESTED** [review]

File: bash.exe.stackdump (hunk 1 above)
Comment (AJLambda) [reply bash.exe.stackdump]: oops, not sure how that got in there , thx ill fix it

**Review by AJLambda — COMMENTED** [review]
Deleted the extra file

**Review by AJLambda — COMMENTED** [review]
@KevinBrack  I think the file should be deleted

**Review by KevinBrack — APPROVED** [review]

**Pull request merged**